src/GraphQL/Types/Schema.cs

```diff
@@ -411,8 +412,45 @@ protected virtual void Validate()
             //TODO: add different validations, also see SchemaBuilder.Validate
             //TODO: checks for parsed SDL may be expanded in the future, see https://github.com/graphql/graphql-spec/issues/653
             // Do not change the order of these validations.
+            CoerceInputTypeDefaultValues();
             SchemaValidationVisitor.Instance.Run(this);
             AppliedDirectivesValidationVisitor.Instance.Run(this);
         }
+
+        /// <summary>
+        /// Coerces input types' default values when those values are GraphQLValue nodes.
+        /// This is applicable when the SchemaBuilder is used to build the schema.
+        /// </summary>
+        protected virtual void CoerceInputTypeDefaultValues()
```

Comment:
@sungam3r Review xml comment?

Reply:
I would wrap GraphQLValue and SchemaBuilder  into `<see>`